Formplayer/testPages/webapps/app_preview_basics.py

```diff
@@ -0,0 +1,204 @@
+        print("refesh button")
+        assert self.is_present_and_displayed(self.toggle_button), "Toggle button is not present"
+
+    def back_button_functionality(self):
+        self.wait_to_click(self.application_menu_id)
+        self.wait_to_click(self.select_test_application)
+        self.wait_to_click(self.view_app_preview)
+        self.wait_to_click(self.start_option)
+        assert not self.is_present_and_displayed(self.start_option)
+        self.wait_to_click(self.back_button)
+        assert self.is_present_and_displayed(self.start_option)
+
+    def refresh_button_functionality_01(self):
+        self.wait_to_click(self.application_menu_id)
+        self.wait_to_click(self.select_test_application)
+        self.wait_to_click(self.view_app_preview)
+        self.wait_to_click(self.start_option)
+        self.wait_to_click(self.case_list_menu)
+        self.wait_to_click(self.refresh_button)
+        assert not self.is_present_and_displayed(self.registration_form)
+        assert self.is_present_and_displayed(self.start_option)
+
+    def refresh_button_functionality_02(self):
+        self.wait_to_click(self.application_menu_id)
+        self.wait_to_click(self.select_test_application)
+        self.wait_to_click(self.view_app_preview)
+        self.wait_to_click(self.form_builder_registration_form)
+        self.wait_to_click(self.add_question)
+        self.wait_to_click(self.text_question_type)
+        self.wait_to_clear_and_send_keys(self.text_question_display_text, "Test")
+        self.wait_to_click(self.save_question_button)
+        self.wait_to_click(self.refresh_button)
+        self.wait_to_click(self.start_option)
+        self.wait_to_click(self.case_list_menu)
+        self.wait_to_click(self.registration_form)
+        assert self.is_present_and_displayed(self.new_test_question)
+        self.wait_to_click(self.formplayer_test_question)
+        self.wait_to_click(self.delete_button)
+        self.wait_to_click(self.save_question_button)
+        self.wait_to_click(self.refresh_button)
+        self.wait_to_click(self.start_option)
+        self.wait_to_click(self.case_list_menu)
+        self.wait_to_click(self.registration_form)
+        assert not self.is_present_and_displayed(self.new_test_question)
+
+    def web_user_submission(self):
+        self.wait_to_click(self.application_menu_id)
+        self.wait_to_click(self.select_test_application)
+        self.wait_to_click(self.view_app_preview)
+        self.wait_to_click(self.start_option)
+        self.wait_to_click(self.case_list_menu)
+        self.wait_to_click(self.registration_form)
+        self.wait_to_clear_and_send_keys(self.name_question, self.name_input)
+        self.wait_to_click(self.dob_question)
+        self.wait_to_click(self.click_today_date)
+        self.wait_to_click(self.close_date_picker)
+        self.wait_to_clear_and_send_keys(self.mobileno_question,"9261437577")
```

Comment:
here as well, use the already created function instead of hard coding the phone number

Reply:
Updated all the hardcoded mobile number with fetch_phone_number function